bundle.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM scratch
```

Comment:
An alternative to building from scratch would be to use a multi-stage build and use `make` targets to get everything ready for building a bundle.